### DISCUSSION TITLE
Finetuner details (methodology and parameters)

### DISCUSSION THREAD

**Discussion by vahuja4** [jina-ai/finetuner#171] [Q&A] [answered]
Hi, where can I read more about the underlying finetuning methodology? And, also about the parameters which are in the 'Advanced' section of the finetuner UI.

**Comment by hanxiao** [comment, marked as answer]
thanks for your interest, right now for the methodology you can refer to https://finetuner.jina.ai/components/tuner/ 
the `Advanced` section parameters are described here https://finetuner.jina.ai/components/labeler/#advanced

That's all we have right now. More docs are coming in the next few weeks, keep an eye on us.
